RevenueCatUI/CustomerCenter/ViewModels/ManageSubscriptionsViewModel.swift

```diff
@@ -98,14 +98,14 @@ class ManageSubscriptionsViewModel: ObservableObject {
         let customerInfo = try await purchasesProvider.customerInfo()
 
         guard let currentEntitlement = customerInfo.earliestExpiringAppStoreEntitlement(),
```

Comment:
this code is not handling correctly the case when both a subscription and a lifetime grant the same entitlement. The issue is that the entitlement will be associated to the lifetime since that's what the backend returns, even though we want to display the monthly information.

We don't have enough information in the CustomerInfo at the moment to filter which subscriptions are Apple subscriptions, so we can't have a better logic for this. When we have that information, we'll be able to filter those out and don't need the entitlement to determine which subscription to show.